WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginListActivity.java

```diff
@@ -134,6 +134,9 @@ private void refreshPluginList() {
     @SuppressWarnings("unused")
     @Subscribe(threadMode = ThreadMode.MAIN)
     public void onSitePluginsFetched(OnSitePluginsFetched event) {
+        if (isFinishing()) {
+            return;
+        }
         mProgressBar.setVisibility(View.GONE);
         if (event.isError()) {
             ToastUtils.showToast(this, "An error occurred while fetching the plugins: "
```

Comment:
The text of this toast is hard-coded.

Reply:
This was meant to be a log 🤦‍♂️ Fixed in d7b2840de93d11ee9c2f8e82b46b9810cbfbb632

This also fixes https://github.com/wordpress-mobile/WordPress-Android/pull/6973#issuecomment-350814887.